### DISCUSSION TITLE
Single File Flac with Cue Sheet Support

### DISCUSSION THREAD

**Discussion by Gluecksrabe** [rern/rAudio#1847] [Fixed Issues]
Hi,

In my library I have a lot of single file flac files of complete CDs or ripped LPs with cue sheets. Unfortunately the support by rAudio is getting worse. 

1. No cover is displayed although a cover.jpg is saved in the same directory as the single file flac and cue sheet. If rAudio thinks it has identified the album (sometimes this goes wrong) it will show a cover but this cannot be saved. 
2. The individual performer (especially important for samplers) is not displayed.
3. Since the last rAudio update the individual tracks are displayed twice (in the example 32 instead of 16 tracks).

I hope that these bugs can be resolved soon. 

Best regards,

Bruno
![Single File Flac with Cue Sheet](https://github.com/user-attachments/assets/e36541d1-1beb-4955-ad1a-cfdb08a4267e)
[Jazz on Christmas.cue.txt](https://github.com/user-attachments/files/18199729/Jazz.on.Christmas.cue.txt)

**Comment by rern** [comment]
Please try the UPDATE.
(Long-press `Update` > `Ok` to branch 'UPDATE')

**Comment by Gluecksrabe** [comment]
Dear rern,

I have updated rAudio and it is nearly perfect now, thank you for the fix. Issue 1 and 3 have been recified. However, in the album view the tracks now seem to be sorted by performer not by track number (**C**het Baker first, **T**he Ramsey Lewis Trio last). When I add this album to the playlist the sort order is correct (according to track number) and the individual performers are also displayed. 

Best regards,

Bruno
![rAudio single file flac with cover displayed](https://github.com/user-attachments/assets/bf28ed5d-4594-49d0-b776-6fa5ab159ff6)
![rAudio single file flac playlist](https://github.com/user-attachments/assets/5ad45ba3-fca1-4cef-b588-576b32bcdfbe)

**Comment by Gluecksrabe** [comment]
I have updated to the latest version and the sort order in album view is now correct according to track number. Thank you rern for also fixing this issue.

The names of individual performers (like in the playlist view) are not yet displayed. However, this is not important but would be a nice feature for a future version.

Bruno
